common/src/main/java/io/netty/util/AsciiString.java

```diff
@@ -1385,6 +1388,19 @@ public static AsciiString of(CharSequence string) {
     }
 
     /**
+     * Returns an {@link AsciiString} containing the given string.
```

Comment:
`containing the given string.` -> `and retains/caches the input string for later use in {@link #toString()}`